### PR TITLE
Remove PoH priority tuning

### PR DESCRIPTION
**Pull request by carllin**
#### Problem
Setting PoH priority is causing performance degradation in the rest of the system, specifically delaying scheduling of other critical functionality including:

1) Reading data off the socket in FetchStage
2) Signature verification
3) Accounts Storage

#### Summary of Changes
Disable PoH priority tuning.

Fixes #

### CODE REVIEW

**Comment by pgarg66** [comment]
@carllin I am trying a few things to keep the tuning, but fix the problem with scheduling. Please wait before you merge this change.

**Comment by pgarg66** [comment]
@carllin we can go ahead with this PR. I was able to fix the spikes in sigverify/snapshots. The spikes in replay/confirmation were still there. So, for time being we can comment out the RT priority.

**Comment by carllin** [comment]
@pgarg66, yeah lets see if this actually fixes those replay spikes in the long term, might just have been flaky when I was testing it :P

File: sys-tuner/src/main.rs [hunk 1]
```diff
@@ -1,69 +1,6 @@
 use clap::{crate_description, crate_name, value_t_or_exit, App, Arg};
 use log::*;
 
-#[cfg(target_os = "linux")]
-fn tune_poh_service_priority(uid: u32) {
```
Comment (pgarg66) [inline sys-tuner/src/main.rs]: @carllin let's not delete the function. We might need it later. We can ignore the unused check for now.

**Comment by solana-grimes** [comment]
:broken_heart: Unable to automerge due to CI failure

**Comment by mvines** [comment]
@carllin / @pgarg66 - do we need an issue to re-enable this feature, or did you conclude that it's not required anymore at all?

**Comment by pgarg66** [comment]
@mvines I'll submit an issue to track it.

**Pull request merged**